crates/ironrdp-graphics/src/color_conversion.rs

```diff
@@ -235,30 +259,40 @@ impl From<YCbCr> for Rgb {
 }
 
 impl From<Rgb> for YCbCr {
+    #[inline(always)]
```

Comment:
question: Can you add a brief comment explaining that this is backed by benchmarks and the performance improvement that was measured if you still have the numbers.

Reply:
It's already inlined by rust by default. not sure if we want to be explicit about it here. My hope is that we can leverage yuvutils some time later anyway.

Reply:
dropped the patch for now